demo/spacy_annotator_demo.ipynb

```diff
@@ -254,7 +254,7 @@
    "id": "b0f48861-f12e-4dd0-bdae-6dc59d17e136",
    "metadata": {},
    "source": [
-    "### Inspect output"
+    "### Inspect output and save dataframe of annotations to .spacy format for training in Spacy3 pipeline."
```

Comment:
I would change it to:   
`## Inspect output and save dataframe of annotations to .spacy format for training in Spacy3 pipeline`